rdrf/rdrf/models/definition/models.py

```diff
@@ -811,6 +812,7 @@ class CommonDataElement(models.Model):
         help_text="The text to use in any public facing questionnaires/registration forms")
 
     important = models.BooleanField(default=False, help_text="Indicate whether the field should be emphasised with a green asterisk")
+    randomize = models.BooleanField(default=False, help_text="Randomize permitted value group value is this CDE has a PVG set")
```

Comment:
typo: 'is this CDE' should read 'if this CDE'

Reply:
Updated.